### ISSUE TITLE
Translation of the Language Tour: French

### ISSUE THREAD

**Issue by usama-elm** [gleam-lang/language-tour#106]
I have decided to embark on the translation of the Language Tour in french but I have faced some choices that might be better discussed at large.

- Neutrality of language: do we keep the translation neutral in a technical way or inclusive using inclusive writing in french
- Use of you. In french there are 2 yous, the polite one (vous) and the informal one (toi)

**Comment by ghivert** [comment]
Hi!

Nice to see the subject here, I'd love to see the language tour in French, and I'd be happy to help on this!
To start, I think it's worth to point at [that video of Scilabus](https://www.youtube.com/watch?v=url1TFdHlSI), because it covers the inclusive writing in French, and in a really nice way.

That being said, I think passive way (to remove subject, and so to avoid at most having to designating a subject) is a nice way to workaround inclusivity. I also prefer to keep "vous" and not "tu", mainly because the former is more polite, and we want to address everyone in the same way, in a respectful way. (Not saying "tu" is disrespectful, but more than it can be received differently according to the reader.)
Personally, I prefer to avoid inclusive writing with the median dot (·), because I find it a little harder to read, while "iels" or "auteurices" are not in my opinion. But I think it's almost always possible to avoid using gendered words, by using periphrasis (périphrases) and non-gendered words (instead of saying "les hommes & les femmes concerné·e·s", we can say "les personnes concernées" for example).

**Comment by adrian-goe** [comment]
I found this issue here as I had a similar idea for German. How could translation work in general? I saw in content, there is a `en.html` file. So for example German, it would just be `de.html`.

**Comment by lpil** [comment]
Yup that, but we would also need some solution for how to edit the inline comments. Perhaps some interpolation syntax which we check at build time.
